config.md

```diff
@@ -308,15 +308,21 @@ The semantics are the same as `Path`, `Args` and `Env` in [golang Cmd](https://g
 
 This OPTIONAL property contains arbitrary metadata for the container.
 This information MAY be structured or unstructured.
-Annotations are key-value maps.
+Annotations MUST be key-value maps.
```

Comment:
@vishh what do you mean by not expecting any structure? People can put json in the value string if they want right?